cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -1130,48 +1054,6 @@ public void testLoadBalancerSourceRangeFromListener() {
         }
     }
 
-    @Test
-    public void testLoadBalancerSourceRangeFromTemplate() {
-        List<String> sourceRanges = new ArrayList();
-        sourceRanges.add("10.0.0.0/8");
-        sourceRanges.add("130.211.204.1/32");
-
-        Kafka kafkaAssembly = new KafkaBuilder(ResourceUtils.createKafka(namespace, cluster, replicas,
-                image, healthDelay, healthTimeout, metricsCm, jmxMetricsConfig, configuration, emptyMap()))
-                .editSpec()
-                    .editKafka()
-                        .withNewListeners()
-                            .addNewGenericKafkaListener()
-                                .withName("external")
-                                .withPort(9094)
-                                .withType(KafkaListenerType.LOADBALANCER)
-                                .withTls(true)
-                            .endGenericKafkaListener()
-                        .endListeners()
-                        .withNewTemplate()
-                            .withNewExternalBootstrapService()
-                                .withLoadBalancerSourceRanges(sourceRanges)
-                            .endExternalBootstrapService()
-                            .withNewPerPodService()
-                                .withLoadBalancerSourceRanges(sourceRanges)
-                            .endPerPodService()
-                        .endTemplate()
-                    .endKafka()
-                .endSpec()
-                .build();
-        KafkaCluster kc = KafkaCluster.fromCrd(kafkaAssembly, VERSIONS);
-
-        // Check external bootstrap service
-        Service ext = kc.generateExternalBootstrapServices().get(0);
-        assertThat(ext.getSpec().getLoadBalancerSourceRanges(), is(sourceRanges));
-
-        // Check per pod services
-        for (int i = 0; i < replicas; i++)  {
-            Service srv = kc.generateExternalServices(i).get(0);
-            assertThat(srv.getSpec().getLoadBalancerSourceRanges(), is(sourceRanges));
-        }
-    }
-
     @Test
     public void testLoadBalancerSourceRangeConflict() {
```

Comment:
I think you can remove this test as well since the conflict does not exist anymore, or?

Reply:
I think this test is still needed as it's part of the `testLoadBalancerSourcesRangeFromListener` and we still wanted to make sure the source ranges exist with the new configuration correct?